src/services/svc-service.ts

```diff
@@ -82,12 +91,44 @@ export class SvcService {
     })
   }
 
-  public startServices(services: string[]): void {
+  public startServices(services: string[]): Promise<ProcessExecutionResult> {
+    this.precheckEnabledServices(services)
+
+    SvcService.debug(`starting service ${services.join(', ')}`)
+
+    return this.processDockerExecution(SERVICE_TYPE.DOCKER_UP, services)
+  }
+
+  public stopServices(services: string[]): Promise<ProcessExecutionResult> {
+    this.precheckEnabledServices(services)
+
+    SvcService.debug(`stopping service ${services.join(', ')}`)
+
+    return this.processDockerExecution(SERVICE_TYPE.DOCKER_STOP, services)
+  }
+
+  private processDockerExecution(
+    serviceType: string,
```

Comment:
Here you can use more specific type: `SERVICE_TYPE`. What about renaming the enum as `ServiceType`? All the other enums we have defined are using camel case. Maybe also `DockerServiceType`, since we have a similar enum in `svc-process.ts`

Reply:
done. thanks for pointing it out